### DISCUSSION TITLE
Control panel from a web browser

### DISCUSSION THREAD

**Discussion by elohim85** [OpenIdentityPlatform/OpenDJ#226] [Q&A] [answered]
Hi all,

Is there any way to access the administration from a web browser or any other tool outside the machine where it is installed?

Help me pls.

Thanks

**Comment by maximthomas** [comment, marked as answer]
Unfortunately, OpenDJ does not have a web admin panel. I suggest you use a third-party tool

**Reply by elohim85** [reply]
Ok thanks, Do you know of any that you can recommend?

Regards

**Reply by maximthomas** [reply]
There is a tool, http://phpldapadmin.sourceforge.net/wiki/index.php/Main_Page, but I don't have any personal usage experience

**Reply by elohim85** [reply]
Very thanks
